catalog/app/components/Collaborators/Table.tsx

```diff
@@ -77,7 +77,7 @@ export default function Table({ collaborators }: TableProps) {
       <M.TableBody>
         {sorted.map(
           ({ collaborator: { email, username }, permissionLevel = 'UNKNOWN' }) => (
-            <M.TableRow>
+            <M.TableRow key={email + username}>
```

Comment:
username is unique, so it should be enough, when email + username *may be* non-unique, e.g. `user@example.com` + `user` vs `user@example.co` + `muser` (or we should at least use a separator)